BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -233,6 +234,15 @@ fileprivate final class ScrollerWrapperView: UIView {
     }
 
     private var refreshAction: () -> Void = { }
+    
+    private var scrollToTopTrigger: ScrollView.Trigger? {
+        didSet {
+            oldValue?.action = { }
+            scrollToTopTrigger?.action = { [weak self] in
+                self?.scrollView.setContentOffset(CGPoint(x: 0, y: 0), animated: false)
```

Comment:
I think we should allow specifying an animation param here; probably by passing it to the `fire()` method.